modules/tests/runner/mod.rs

```diff
@@ -63,7 +63,7 @@ impl IbcTestRunner {
             max_history_size,
             Height::new(Self::revision(), initial_height),
         );
-        assert!(self.contexts.insert(chain_id, ctx).is_none());
+        self.contexts.insert(chain_id, ctx);
```

Comment:
I believe this assert fails in the new version because the `IbcTestRunner` is shared across tests: `run_tla_steps` now takes a `&mut`, reusing it on all tests, while before `run` took an owned value that was cloned for each test. (This probably means that the assert only fails when the second test is run.)

Reply:
Is it important? Does it need to go back in and be fixed?

Reply:
In my view this is fine given the current behaviour of modelator. But I think that the behaviour in the previous version is more intuitive as it allows this kind of asserts. (In this case we're saying that each chain should be initialized only once, which is true within a single test, but false across tests).

Reply:
Ok, leaving the assert out